### PR TITLE
Remove unused confirm_remove_special code

### LINKED ISSUE #5844: The message settings.special_users.new.confirm_remove_special_user appears to be unused
### What is happening?
I noticed that the message settings.special_users.new.confirm_remove_special_user may be problematic: it looks like it's supposed to have a parameter (probably a username), but it doesn't.

However, when I searched for it in the code, I found nothing. It appears to be used only in translation files and not in code.

If it's not actually used, it should be removed from the English messages file.

### PR DESCRIPTION
**Pull request by osujipius**
NOTE: Please review the pull request process before opening your first PR: https://github.com/WikiEducationFoundation/WikiEduDashboard/blob/master/CONTRIBUTING.md#pull-request-process

## What this PR does
Cleans up unused code in the codebase
Fixes #5844

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -1276,7 +1276,7 @@ en:
         form_label: Grant Special User privileges
         form_placeholder: Username
         confirm_add_special_user: "Add Special User"
-        confirm_remove_special_user: "Remove Special User privileges for"
+        _user: "Remove Special User privileges for"
```
Comment (ragesoss) [inline config/locales/en.yml:1279]: What is this for?

Comment (osujipius) [reply config/locales/en.yml:1279]: Looks like it is unused as well @ragesoss

Comment (ragesoss) [reply config/locales/en.yml:1279]: Oh, I guess it was a typo and you intended to remove the whole line?

Comment (osujipius) [reply config/locales/en.yml]: > Oh, I guess it was a typo and you intended to remove the whole line?

Exactly!
Removed it completely now

**Comment by ragesoss** [comment]
Please exclude formatting changes (such as replacing one type of quote mark with another) to any of the locale files except for `en` and `qqq`, as these files are maintained via translatewiki.net.

File: config/locales/et.yml [hunk 2]
```diff
@@ -1300,4 +1299,4 @@ et:
     assignment_status: Õppeülesanne (%{status})
   multi_wiki:
     selector_placeholder: Alusta vikidomeeni sisestamist
-...
+...
```
Comment (ragesoss) [inline config/locales/et.yml:1302]: Please exclude the newline changes at the end of the locale files.

File: db/schema.rb [hunk 3]
```diff
@@ -11,15 +11,25 @@
 # It's strongly recommended that you check this file into your version control system.
 
 ActiveRecord::Schema[7.0].define(version: 2024_05_15_163420) do
-  create_table "alerts", id: :integer, charset: "utf8mb4", force: :cascade do |t|
+  create_table "admin_course_notes", charset: "utf8mb3", collation: "utf8mb3_general_ci", force: :cascade do |t|
```
Comment (ragesoss) [inline db/schema.rb:14]: No schema or yarn.lock changes should be included.

File: config/locales/et.yml (hunk 2 above)
Comment (osujipius) [reply config/locales/et.yml:1302]: I think this is as a result of taking out the unused code some lines above or do you mean the "..." should be removed entirely?

Comment (ragesoss) [reply config/locales/et.yml:1302]: No, this is likely a result of your text editor automatically remove the invisible newline character at the end of the file.

When you run `git diff`, the only changes should be the lines you removed.

**Pull request merged**